docs/tools/vega-wallet/cli-wallet/latest/changes-in-0-14.md

```diff
@@ -0,0 +1,30 @@
+---
+sidebar_position: 2
+title: Breaking change when upgrading to 0.14
+hide_title: false
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+From version 0.14 onwards, you will need to approve transactions with your Vega Wallet, through the command line. This security measure ensures that only someone with direct access to the wallet can allow transactions to be sent.
+ 
+**Below, find an explanation of changes between the versions.**
+
+:::info
+It's recommended that you keep your wallet app updated to the latest version. 
```

Comment:
```suggestion
It's highly recommended that you keep your wallet app updated to the latest version. 
```

Also, I think we should mention that some operation may fail due to transaction format changes.